draft-ietf-rats-corim.md

```diff
@@ -995,19 +990,23 @@ the sensitive values in memory are encrypted.
 Raw value measurements are typically vendor defined values that are checked by Verifiers
 for consistency only, since the security relevance is opaque to Verifiers.
 
-There are two parts to a `raw-value-group`, a measurement and an optional mask.
+A `raw-value` measurement, or an Endorsement, is a tagged value of type `bytes`.
+This specification defines tag #6.560.
 The default raw value measurement is of type `tagged-bytes` ({{sec-common-tagged-bytes}}).
-Additional raw value types can be defined, but must be CBOR tagged so that parsers can distinguish
-between the various semantics of type values.
 
+Additional value types can be added to `$raw-value-type-choice`, these additional values must be CBOR tagged `bstr`s.
```

Comment:
When you say "can be added", do you mean that a base CoRIM is permitted to have any tagged byte string as registered in https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml ? That would be convenient for me (and Ned, I think). If you mean that any profile extensions to `$raw-value-type-choice` are restricted to specify only tagged byte strings, that's less appealing.